docs/spec/protocol-modules/storage/assignment.md

```diff
@@ -60,15 +40,17 @@ The protocol requires that chunk lengths are sufficiently small that operators w
 We require that the chunk size $C$ satisfy
 
 $$
-C \le \text{NextPowerOf2}\left(\frac{B}{\gamma}\max\left(\frac{\min_jS_j}{\sum_jS_j}, \alpha \right) \right)
+C \le \text{NextPowerOf2}\left(\frac{B}{\gamma}\max\left(\frac{\min_jS_j}{\sum_jS_j}, \frac{1}{M_\text{max}} \right) \right)
 $$
 
 
-where $\gamma = \beta-\alpha$, with $\alpha$ and $\beta$ as defined in the [Storage Overview](./overview.md) and $\alpha = 1/8192$ is a system parameter.
+where $\gamma = \beta-\alpha$, with $\alpha$ and $\beta$ as defined in the [Storage Overview](./overview.md).
+
+This means that as long as an operator has a stake share of at least $1/M_\text{max}$ of encoded data that they will receive will be within a factor of 2 of their share of stake. Operators with less than $1/M_\text{max}$ of stake will receive no more than a $1/M_\text{max}$ of the encoded data. $M_\text{max}$ represents the maximum number of chunks that the disperser can be required to encode per blob. This limit is included because proving costs scale somewhat super-linearly with the number of chunks. 
```

Comment:
nit: the `of` after Mmax should be `, the`?

btw, this is much clear regarding C and M!

Reply:
Thanks